sktime/forecasting/tests/test_all_forecasters.py

```diff
@@ -1024,7 +1024,7 @@ def test_global_forecasting_multiindex(self, estimator_instance):
         # check consistency of forecast hierarchy with training data
         assert set(y_pred.index).issubset(X_test.index)
 
-    @pytest.mark.parametrize("n_columns", (1, 10))
+    @pytest.mark.parametrize("n_columns", (1, 10))  # noqa: PT007
```

Comment:
What is this? If you use `[1, 10]`, will it be solved?

Reply:
It was the wrong value being passed to pytest.mark.parametrize